gapic/api/artman_bigtable.yaml

```diff
@@ -1,5 +1,6 @@
 common:
-  api_name: google-cloud-bigtable-v2
+  short_name: bigtable
+  version: v2
```

Comment:
`version` -> `api_version` makes sense to me because we need to distinguish API/client lib version. Done on the artman PR.

`api_short_name` seems verbose given that this is actually now the only API name in the artman config; how about just calling this `api_name` now? (To be distinguished from [the API `title` in the service config](https://github.com/googleapis/googleapis/blob/master/google/logging/logging.yaml#L6).)